src/static/css/_calls-to-action.css

```diff
@@ -0,0 +1,125 @@
+.calls-to-action--insight {
+    --calls-to-action-bg: var(--color-blue-600);
+    --calls-to-action-border: var(--color-blue-700);
+    --calls-to-action-hover-bg: var(--color-blue-700);
+    --calls-to-action-hover-border:
+        linear-gradient(
+            90deg,
+            var(--fl-linkColor, var(--color-blue-600)) 0%,
+            var(--fl-linkColor, var(--color-blue-100)) 100%
+        );
+    --calls-to-action-focus-bg: var(--color-blue-100);
+    --calls-to-action-focus-border: var(--color-blue-600);
+    --calls-to-action-color: var(--color-white);
+    --calls-to-action-focus-color: var(--color-black);
+}
+
+.calls-to-action--technique {
+    --calls-to-action-bg: var(--color-red-500);
+    --calls-to-action-border: var(--color-red-600);
+    --calls-to-action-hover-bg: var(--color-red-600);
+    --calls-to-action-hover-border: linear-gradient(90deg, var(--color-red-500) 0%, var(--color-red-100) 100%);
+    --calls-to-action-focus-bg: var(--color-red-100);
+    --calls-to-action-focus-border: var(--color-red-500);
+    --calls-to-action-color: var(--color-black);
+    --calls-to-action-focus-color: var(--color-black);
+}
+
+.calls-to-action--practice {
+    --calls-to-action-bg: var(--color-green-500);
+    --calls-to-action-border: var(--color-green-600);
+    --calls-to-action-hover-bg: var(--color-green-600);
+    --calls-to-action-hover-border: linear-gradient(90deg, var(--color-green-500) 0%, var(--color-green-300) 100%);
+    --calls-to-action-focus-bg: var(--color-green-100);
+    --calls-to-action-focus-border: var(--color-green-500);
+    --calls-to-action-color: var(--color-black);
+    --calls-to-action-focus-color: var(--color-black);
+}
+
+.calls-to-action--activity {
+    --calls-to-action-bg: var(--color-orange-500);
+    --calls-to-action-border: var(--color-orange-600);
+    --calls-to-action-hover-bg: var(--color-orange-600);
+    --calls-to-action-hover-border: linear-gradient(90deg, var(--color-orange-500) 0%, var(--color-orange-100) 100%);
+    --calls-to-action-focus-bg: var(--color-orange-100);
+    --calls-to-action-focus-border: var(--color-orange-500);
+    --calls-to-action-color: var(--color-black);
+    --calls-to-action-focus-color: var(--color-black);
+}
+
+.calls-to-action {
+    background: var(--fl-bgColor, var(--calls-to-action-bg, var(--color-blue-600)));
```

Comment:
If you set the variables for the default as noted above, you can remove the third fallback on all of these.